common/sgx/tcbinfo.c

```diff
@@ -537,6 +692,15 @@ oe_result_t oe_parse_tcb_info_json(
                 "Platform TCB (%d) is not up-to-date",
                 platform_tcb_level->status);
         }
+
+        // Display any advisory IDs as warnings
+        if (platform_tcb_level->advisory_ids_size > 0)
+        {
+            OE_TRACE_WARNING(
```

Comment:
>OE_TRACE_WARNING [](start = 12, length = 16)

Feels like a verbose trace. Also, there are 2 "%d"s but only one integar argument.

Reply:
Good catch! thank you.